src/cascadia/TerminalConnection/AzureConnection.h

```diff
@@ -95,9 +92,16 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
 
         std::optional<std::wstring> _ReadUserInput(InputMode mode);
 
-        web::websockets::client::websocket_client _cloudShellSocket;
+        winrt::Windows::Web::Http::HttpClient _httpClient{ nullptr };
+        wil::unique_winhttp_hinternet _socketSessionHandle{};
+        wil::unique_winhttp_hinternet _socketConnectionHandle{};
+        wil::unique_winhttp_hinternet _webSocket{};
 
-        static std::optional<utility::string_t> _ParsePreferredShellType(const web::json::value& settingsResponse);
+        til::u8state _u8State{};
+        std::wstring _u16Str{};
+        std::array<char, 4096> _buffer{};

```

Comment:
I'd remove the `{}`. No need to eagerly initialize 4kB IMO. The other members above technically don't need the `{}` either, but that's more of a matter of taste (I forgot the proper idiom in tech).

Reply:
Fair. I'll drop all 3. ConptyConnection uses this as well, which is where I copied it from

Reply:
I trusted youuuuuuuuuuuuuuuuuuu

         C:\a\_work\1\s\src\inc\til\u8u16convert.h(37): error C26495: Variable 'til::u8state::partials' is uninitialized. Always initialize a member variable (type.6). [C:\a\_work\1\s\src\cascadia\TerminalConnection\TerminalConnection.vcxproj]
         C:\a\_work\1\s\src\cascadia\TerminalConnection\AzureConnection.h(21): error C26495: Variable 'winrt::Microsoft::Terminal::TerminalConnection::implementation::AzureConnection::_buffer' is uninitialized. Always initialize a member variable (type.6). [C:\a\_work\1\s\src\cascadia\TerminalConnection\TerminalConnection.vcxproj]